perllib/FixMyStreet/Cobrand/Bexley.pm

```diff
@@ -139,6 +139,11 @@ sub open311_extra_data_include {
         }
     }
 
+    # Add private comments field
+    push @$open311_only,
+        { name => 'private_comments', description => 'Private comments',
+          value => $row->get_extra_metadata('private_comments') };
```

Comment:
This needs to default to the empty string if not present, otherwise it looks lke Open311.pm warns every time a private comment is undefined.